app/templates/frameworks/dashboard.html

```diff
@@ -23,38 +33,43 @@
 
   {% with messages = get_flashed_messages(with_categories=True) %}
     {% for category, message in messages %}
+    {% endfor %}
+  {% endwith %}
```

Comment:
This `with` block doesn't do anything with the flashed messages - looks like a partial duplicate of the block below and this top one can be deleted.